actions/image_partition_action.go

```diff
@@ -613,6 +620,12 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 			}
 		}
 
+		if i.PartitionType == "gpt" {
```

Comment:
remove outer if statement

Reply:
Squashed the commits and updated the commit. Thanks.